tools/docker-compose/compose.yaml

```diff
@@ -73,6 +73,9 @@ services:
       - ANSIBLE_AI_ENABLE_ONE_CLICK_TRIAL=${ANSIBLE_AI_ENABLE_ONE_CLICK_TRIAL}
       - ANSIBLE_AI_ENABLE_ONE_CLICK_DEFAULT_MODEL_ID=${ANSIBLE_AI_ENABLE_ONE_CLICK_DEFAULT_MODEL_ID}
       - ANSIBLE_AI_ENABLE_ONE_CLICK_DEFAULT_API_KEY=${ANSIBLE_AI_ENABLE_ONE_CLICK_DEFAULT_API_KEY}
+      - CHATBOT_URL=${CHATBOT_URL}
+      - CHATBOT_DEFAULT_PROVIDER=${CHATBOT_DEFAULT_PROVIDER}
+      - CHATBOT_DEFAULT_MODEL=${CHATBOT_DEFAULT_MODEL}
```

Comment:
we dont want to add these in the default env in the make file right? we want the chatpbot env to be optional for wisdom service I guess.

Reply:
Right.  It is an optional feature. I think we need to have an internal doc how to configure for the development purpose, though.